README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
+to verify and configure your devices.
+
+Verify and configure devices on **NI MAX**:
+
+.. image:: max_device_name.png
```

Comment:
The README.rst is included in the Python package itself and uploaded to PyPI and ReadTheDocs. As proposed in this PR, the images will be broken links everywhere but GitHub:

<img width="1096" alt="Screenshot 2024-06-10 at 9 33 18 PM" src="https://github.com/ni/nidaqmx-python/assets/17914399/b95c4c2d-1301-4236-9609-8fe8376df3b7">

Actually, they're broken on GitHub too, at least in this PR:

<img width="1434" alt="Screenshot 2024-06-10 at 9 34 28 PM" src="https://github.com/ni/nidaqmx-python/assets/17914399/27e59db1-d4f9-4bf8-b61b-94e7a21920a0">

The answers to this SO question may help: https://stackoverflow.com/questions/41983209/how-do-i-add-images-to-a-pypi-readme-that-works-on-github